src/pages/ProfilePage.vue

```diff
@@ -167,20 +185,39 @@
         </div>
       </q-card>
     </div>
+    <EditUserModal
+      v-if="openEditModal"
+      :open-modal="openEditModal"
+      :user-id="openIdForEditing"
+      @close="closeEditModal"
+    />
   </q-page>
 </template>
 
 <script setup lang="ts">
 import { User } from 'src/types'
 import { ref } from 'vue'
+import { useUserStore } from 'src/stores/userStore'
+import { useUserActions } from 'src/components/UsersPage/userActions'
+import EditUserModal from 'src/components/modals/EditUserModal.vue'
 
+const userStore = useUserStore()
+const { id } = userStore.$state
+const { username } = userStore.$state
+const { email } = userStore.$state
```

Comment:
не надо дестрактить, и обращайся сразу к полям, $state - это для лохов которые с вью 2 только перешли
а если дестрактишь - то давай вот так:
```suggestion
const { id, username, email } = userStore
```

Reply:
дестрактить заставили... исправляли автоматически...... окей поправлю на последний вариант выглядит хорошо